README.md

```diff
@@ -99,11 +94,9 @@ Valid properties for meshes specification referenced by `mesh_url`:
 
 * RID - (string) The ID of the Mesh.
 * url - (string) The location where the object data resides for this mesh
-* link - (object) An object containing a URL and label description of the mesh
-    * Example ```{"url": "http://example.com", "label": "Mesh URL"}
-* anatomy - (string) The label for the anatomy this mesh describes
-* anatomy_id - (string) The id for the anatomy term (optional)
-* description - (string) A description of the Mesh
+* anatomy - (string) Name of the anatomy this mesh points
```

Comment:
add. "If given, the value for `anatomy` will be used as the display label for the mesh."